### DISCUSSION TITLE
D1 mini PiR Dth22 2 switch and ws2812

### DISCUSSION THREAD

**Discussion by lukaszwitkowski** [RavenSystem/esp-homekit-devices#1799] [MEPLHAA Scripts]
Hello i make multi-function homekit d1 mini.
I have
2 switch with relay
1 dth22 - temperature and humidiflter
1 pir - motion sensor
1 ws2812 - to background wall switch
Powered by 230/5v power adapter
And here i have question
Why my json config add bridge?
Where i can remove  that ?

![image](https://user-images.githubusercontent.com/36861888/159164787-5c4d6906-e44d-41b0-9a02-e65495fd6cdb.jpeg)
![image](https://user-images.githubusercontent.com/36861888/159164801-81adf779-d77b-4e7d-84d0-5d73693f1930.jpeg)
![image](https://user-images.githubusercontent.com/36861888/159164814-07779889-a019-4f7d-af01-d532f13a013f.jpeg)

My json:

{
  "c": { "l": 13 },
  "a": [{
    "t": 1,
    "0": { "r": [{ "g": 0, "v": 0  }] },
    "1": { "r": [{ "g": 0, "v": 1 }] },
    "b": [{"g":5,"p":1,"t":1},{"g":5,"t":0}],
    "s": 5
  },
  {
    "t": 1,
    "0": { "r": [{ "g": 3, "v": 0  }] },
    "1": { "r": [{ "g": 3, "v": 1 }] },
    "b": [{"g":4,"p":1,"t":1},{"g":4,"t":0}],
    "s": 5
  },
  {
    "t":12,"i":10, 	"0":{"r":[{"g":2,"v":0}]},
  "1":{"r":[{"g":2,"v":1}]},
    "f1":[{"g":16}]},
{
    "t":30,
    "ty":8,
    "g":[1,1,20]
  },
{
    "t": 24,
    "g": 14,
    "n": 2,
    "j": 30,
    "z": -0.8
  }]
}

**Comment by mariusvasss** [comment]
When you have more then 4 devices it's a bridge...
